data/card_catalogue/unique-1.json

```diff
@@ -177,20 +177,20 @@
         "active": "Summon a rigged pair of dice which you can control with your mind.",
         "passive": "Whenever you would roll a d20, flip a coin instead. Heads for a critical success, tails for a critical failure." 
       },
-      "flavour": "Whomever lives, gambles with life.",
+      "flavour": "",
       "image": "res/art/illustration/The Ultimatum.png"
     }
   },
   {
     "title": "Where-Ware Charm",
     "upgrade": null,
-    "tier": "blue",
+    "tier": "green",
     "details": {
       "rules": {
-        "active": "Learn the current location of an item you've touched in the past.",
-        "passive": null
+        "active": "Teleport to a visible object you've touched",
+        "passive": "You know the location of every object you've ever touched"
       },
-      "flavour": "At last, the location of the missing socks.",
+      "flavour": "I don't know who stole my socks from the dryer, but I do know their days are numbered.",
```

Comment:
Where-Ware wasn't a proper blue by our heuristics because it wasn't a distraction. I liked the left-field flavour of it, so I upgraded it into a green card.